tensorboard/webapp/feature_flag/store/feature_flag_metadata.ts

```diff
@@ -37,6 +37,9 @@ export type FeatureFlagMetadata<T> =
       // Additionally they should specify a way to parse the query param string
       // values into the feature flag value.
       parseValue: (str: string) => T;
+      // Some overridable feature flags should be sent to the server if the user
+      // has specified an override value.
+      sendToServer?: boolean;
```

Comment:
nitpick: `sendToServer` makes me think it will always be sent to the server, not just when it is overridden.
```suggestion
      sendToServerWhenOverridden?: boolean;
```

Reply:
Done.